Stubs/Calendar.astub

```diff
@@ -0,0 +1,133 @@
+import org.checkerframework.checker.index.qual.NonNegative;
```

Comment:
I thought `java.util.Calendar` was annotated in the JDK (https://github.com/typetools/checker-framework/blob/3c427f380f61f234688893467d9bc433cf1a584c/checker/jdk/index/src/java/util/Calendar.java). Why are those annotations insufficient?

Reply:
First, I didn't suspect the imprecision in double dimensional array, and hence, I thought the error was being produced because `java.util.Calendar` was not annotated, but now I will rectify it.